lib/pacemaker/pcmk_output.c

```diff
@@ -66,20 +66,15 @@ colocations_header(pe_resource_t *rsc, rsc_colocation_t *cons,
     return retval;
 }
 
-PCMK__OUTPUT_ARGS("colocations-list", "pe_resource_t *", "gboolean", "gboolean")
+PCMK__OUTPUT_ARGS("colocations-list", "pe_resource_t *", "gboolean")
 static int colocations_list(pcmk__output_t *out, va_list args) {
     pe_resource_t *rsc = va_arg(args, pe_resource_t *);
-    gboolean dependents = va_arg(args, gboolean);
     gboolean recursive = va_arg(args, gboolean);
 
     GList *lpc = NULL;
     GList *list = rsc->rsc_cons;
```

Comment:
Since we're splitting the functions, we don't need the list variables anymore

Reply:
Fixed, will show up in the next push (whenever that is).